core/vm/contract.go

```diff
@@ -58,6 +58,9 @@ type Contract struct {
 
 	Gas   uint64
 	value *uint256.Int
+
+	// zkevm
+	IsCreate bool
```

Comment:
contract_zkevm?

Reply:
I can take a look but I think this would create wider spread changes rippling through the codebase.  I know we want to avoid clashes with upstream but there is a one liner to set this in the contract create flow otherwise it's left to default as false

Reply:
fair enough